test/runtests.jl

```diff
@@ -0,0 +1,70 @@
+using LambdaMaker
+using Test
+
+SAMPLE_FILENAME = "Dockerfile"
+
+@testset "_copy_file" begin
+    mktempdir() do RESOURCE_DIR
+        file_path = joinpath(RESOURCE_DIR, SAMPLE_FILENAME)
+
+        @testset "dest file does not exist" begin
+            LambdaMaker._copy_file(RESOURCE_DIR, SAMPLE_FILENAME)
+            @test isfile(file_path)
+        end
+
+        @testset "dest file exists" begin
+            @test_throws ArgumentError LambdaMaker._copy_file(RESOURCE_DIR, SAMPLE_FILENAME)
+        end
+
+        @testset "force=true" begin
+            LambdaMaker._copy_file(RESOURCE_DIR, SAMPLE_FILENAME; force=true)
+            @test isfile(file_path)
+        end
+    end
+end
+
+@testset "_render_file" begin
+    mktempdir() do RESOURCE_DIR
+        render_args = Dict("PKG" => "foobar")
+
+        LambdaMaker._render_file(SAMPLE_FILENAME, RESOURCE_DIR, render_args)
+
+        result_file_path = joinpath(RESOURCE_DIR, SAMPLE_FILENAME)
+        @test isfile(result_file_path)
```

Comment:
Maybe check for the rendered `CMD [ "foobar.handle_event"]` in the resulting file just to be safe?